packages/schematics/migrations/update-7-7-0/update-7-7-0.spec.ts

```diff
@@ -55,7 +55,6 @@ describe('Update 7.7.0', () => {
           },
           resolver: '@nrwl/builders/plugins/jest/resolver',
           moduleFileExtensions: ['ts', 'js', 'html'],
-          collectCoverage: true,
```

Comment:
This is fine..

But we should also create an update for `8.3.0` which will remove the collectCoverage support...

Something like search for AST Nodes which is a property assignment for "collect coverage"

Reply:
Sounds good, I will get started on this.

Reply:
@FrozenPandaz I have added an 8.3.0 migration that will change the `collectCoverage` to `false` if it is found and currently set as `true`. I didn't see any migrations yet for 8.3.0, so let me know if there is a better place to do this.

Either way, should be ready for your review